common/utils/array-utils.js

```diff
@@ -13,6 +18,20 @@ function insertIf(condition, ...elements) {
   return condition ? elements : [];
 }
 
+/**
+ * @description used to identify if an array is filled with real values
+ * @export
+ * @param {array} potentialArray passed arg isnt required to be an array
+ * @returns {boolean}
+ */
+function isFilledArray(potentialArray) {
+  return Boolean(
+    Array.isArray(potentialArray) &&
+      !isEmpty(potentialArray) &&
```

Comment:
Since we already know it's an array could we just use `... && potentialArray.length && ...`?